Library/ViewModels/ProjectPamphletMainCellViewModelTests.swift

```diff
@@ -708,6 +770,34 @@ final class ProjectPamphletMainCellViewModelTests: TestCase {
   }
 
   // swiftlint:disable line_length
+  func testCreatorButtonHidden_Control() {
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~ [
+        OptimizelyExperiment.Key.nativeProjectPageConversionCreatorDetails.rawValue: OptimizelyExperiment.Variant.control.rawValue
+      ]
+
+    withEnvironment(optimizelyClient: optimizelyClient) {
+      self.vm.inputs.configureWith(value: (.template, nil, (nil, true)))
+      self.vm.inputs.awakeFromNib()
+
+      self.creatorButtonIsHidden.assertValues([false])
```

Comment:
We can move these tests into the `testCreatorDetailsVisibility_NilCreatorDetails` and `testCreatorDetailsVisibility_NonNilCreatorDetails` tests below to just test all the UI visibility together.